setup.py

```diff
@@ -20,14 +20,14 @@
 ]
 
 setup(
-    name='pyjose',
+    name='jose',
```

Comment:
But we're wrapping jose. And the namespace is already collided. And none of the other packages are in a distro, so we'll be the first. We also implement the complete specs, which no other project does. I have aspirations to be the reference implementation. :)